Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.6-onbuild
+
+RUN ./script/generatecert
+EXPOSE 8080
+
+CMD app \
+  -addr=localhost:8080 \
```

Comment:
Actually, just `-addr=:8080` will work as well. https://golang.org/pkg/net/#Listen

>If host is omitted, as in ":8080", Listen listens on all available interfaces instead of just the interface with the given host address.